### DISCUSSION TITLE
I have completed my profile with name and short bio, but still can't be verified

### DISCUSSION THREAD

**Discussion by chunfenri** [community/community#53097] [GitHub Education] [answered]
<img width="620" alt="image" src="https://user-images.githubusercontent.com/47965870/232705650-6bae09b5-7123-4c8d-976f-af2851190213.png">

**Comment by chunfenri** [comment]
<img width="257" alt="image" src="https://user-images.githubusercontent.com/47965870/232711057-f2207b76-2401-4bc3-a92d-3360e4575fd5.png">
Same Issue tried 8 times 😢😢😢

**Reply by ps2109** [reply]
yeah same is with me 
did you got any resolution from anyone yet 
please do share and help me here as well in english  

Thanks

**Comment by chunfenri** [comment, marked as answer]
Finally success, TIPS for Chinese Students:
1.不要用vpn
2.我用了学信网的学籍认证报告，中文和翻译软件翻译的英文（百度翻译的网页就行，一定要放大到可以识别）并排放
3.用手机拍摄，类别选择other，自己输入official verification report from the ministry of education

1. Do not use VPN
2. I used the official verification report from Xuexin Network. I was using both Chinese web and translation software (Baidu Translate's webpage is sufficient, make sure to enlarge it to be recognizable) inside one shot
3. Use your phone to capture, select the category 'other', and enter the “official verification report from the registry of education ”yourself

**Reply by ps2109** [reply]
it did not help me i am still getting same response  as mentioned by @chunfenri 
can someone help me here in english  ?

**Comment by ghost** [comment]
我也是这个个人资料验证，过不去，你最后怎么通过的啊？

**Reply by chunfenri** [reply]
你得先填写好个人信息，姓名和bio，如果还有问题就是机器在照片里没有识别到相应的字段（学校、姓名的英文名），拍清楚点就行，你就用我的那一套走就行了
